settings.go

```diff
@@ -9,7 +9,7 @@ const (
 	defaultHeaderTableSize   uint32 = 4096
 	defaultConcurrentStreams uint32 = 100
 	defaultWindowSize        uint32 = 1048896
-	defaultDataFrameSize     uint32 = 1048896
+	defaultDataFrameSize     uint32 = 1<<14
```

Comment:
Not that I care that much about this, but windowSize still have that constant.

Reply:
I didn't handle this one yet but I can. The default value will be 2^16
```
SETTINGS_INITIAL_WINDOW_SIZE (0x4):
    Indicates the sender's initial window size (in octets) for stream-level flow control. The initial value is 2^16-&(65,535) octets.
```